python_modules/dagster/dagster/core/types/loadable_target_origin.py

```diff
@@ -31,26 +31,10 @@ def __new__(
         )
 
     def get_cli_args(self):
-
-        # Need to ensure that everything that consumes this knows about
-        # --empty-working-directory and --use-python-package
         args = (
-            (
-                (
-                    [
-                        "-f",
-                        self.python_file,
-                    ]
-                    + (
-                        ["-d", self.working_directory]
-                        if self.working_directory
-                        else ["--empty-working-directory"]
```

Comment:
this is also significant - instead of setting --empty-working-directory when no working dir is explicitly set in the workspace.yaml, we set no argument, which will cause the gRPC server to use os.getcwd() as the default working directory. That working dir will get passed down into load_python_file / load_python_module and used to do the import.